forbidigo uses makezero in the lint reports

Thank you for creating the issue!

- [x] Yes, I'm using a binary release within 2 latest major releases. Only such installations are supported.
- [x] Yes, I've searched similar issues on GitHub and didn't find any.
- [x] Yes, I've included all information below (version, config, etc).

I enabled `forbidigo` and got these errors:

```
internal/i18n/main.go:15:2: use of `fmt.Println` forbidden by pattern `^fmt\.Print(|f|ln)$` (makezero)
        fmt.Println(string(i18n.GenerateJSON()))
        ^
pkg/cmd/gen/i18n/i18n.go:17:2: use of `fmt.Println` forbidden by pattern `^fmt\.Print(|f|ln)$` (makezero)
        fmt.Println(string(i18n.GenerateJSON()))
        ^
pkg/cmd/gen/config/config.go:55:2: use of `fmt.Println` forbidden by pattern `^fmt\.Print(|f|ln)$` (makezero)
        fmt.Println(string(data))
        ^
```

As you can see It uses `makezero` in the report

Issue was introduced in here: https://github.com/golangci/golangci-lint/pull/1569/files#diff-f0557183cd3a50b72b0c7c3477c73b79c2e8bdd805652fe7c234cc23bf661b95R49

the `forbidigoName` should be used instead of `makezeroName`

--- a/pkg/golinters/forbidigo.go
+++ b/pkg/golinters/forbidigo.go
@@ -12,18 +12,17 @@ import (
 	"github.com/golangci/golangci-lint/pkg/result"
 )
 
-const forbidigoName = "forbidigo"
-
 func NewForbidigo() *goanalysis.Linter {
+	const linterName = "forbidigo"
 	var mu sync.Mutex
 	var resIssues []goanalysis.Issue
 
 	analyzer := &analysis.Analyzer{
-		Name: forbidigoName,
+		Name: linterName,
 		Doc:  goanalysis.TheOnlyanalyzerDoc,
 	}
 	return goanalysis.NewLinter(
-		forbidigoName,
+		linterName,
 		"Forbids identifiers",
 		[]*analysis.Analyzer{analyzer},
 		nil,
@@ -34,7 +33,7 @@ func NewForbidigo() *goanalysis.Linter {
 			var res []goanalysis.Issue
 			forbid, err := forbidigo.NewLinter(s.Forbid)
 			if err != nil {
-				return nil, errors.Wrapf(err, "failed to create linter %q", forbidigoName)
+				return nil, errors.Wrapf(err, "failed to create linter %q", linterName)
 			}
 
 			for _, file := range pass.Files {
@@ -46,7 +45,7 @@ func NewForbidigo() *goanalysis.Linter {
 					res = append(res, goanalysis.NewIssue(&result.Issue{
 						Pos:        hint.Position(),
 						Text:       hint.Details(),
-						FromLinter: makezeroName,
+						FromLinter: linterName,
 					}, pass))
 				}
 			}

